config/config.go

```diff
@@ -16,6 +16,7 @@ type Config struct {
 	HealthCheckCriticalTimeout time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
 	OutputFilePath             string        `envconfig:"OUTPUT_FILE_PATH"`
 	KafkaConfig                KafkaConfig
+	ContentUpdatedTopicFlag    bool `envconfig:"CONTENT_UPDATED_TOPIC_FLAG"`
```

Comment:
Also, can you reorder this in alphabetical order as well please? It's a good practice to start off with (especially on this new app) and makes it easier to find things